### DISCUSSION TITLE
Failed Attach Longhorn volume

### DISCUSSION THREAD

**Discussion by northboys** [strimzi/strimzi-kafka-operator#6106] [Q&A]
hi, i got an error while deploying kafka cluster using longhorn as storage class
<img width="1680" alt="Screen Shot 2021-12-24 at 10 02 59" src="https://user-images.githubusercontent.com/38155944/147311353-8dd1b9d9-24ad-496e-a477-24ff244f12d0.png">
This is the yaml configuration that I have used.
![Screen Shot 2021-12-24 at 10 05 03](https://user-images.githubusercontent.com/38155944/147311452-33d5fb9c-a25c-4214-b16c-6e0e2fb47e87.png)

**Comment by scholzj** [comment]
Sounds similar to #6098. Attaching and mounting the volumes is not something Strimzi does. It is something Kubernetes and Longhorn are responsible for. So I guess you should check their logs and maybe they tell you more about the issue. So I think you need to check there to find out what the problem is.
